rows.go

```diff
@@ -127,6 +129,103 @@ type Rows struct {
 	closeErr  error
 }
 
+// new Rows and add a set of driver.Value by using the struct reflect tag
+func newRowsFromStruct(m interface{}, tagName ...string) (*Rows, error) {
+	val := reflect.ValueOf(m).Elem()
+	num := val.NumField()
+	if num == 0 {
+		return nil, errors.New("no properties available")
+	}
+	columns := make([]string, 0, num)
+	var values []driver.Value
+	tag := "json"
+	if len(tagName) > 0 {
+		tag = tagName[0]
+	}
+	for i := 0; i < num; i++ {
+		f := val.Type().Field(i)
+		column := f.Tag.Get(tag)
```

Comment:
This code allows to use JSON tags, but doesn't fully support them.

Examples of incorrectly handled values: "-", "data,omitempty".

I think that using JSON tags is a bad default.